cropharvest/eo/eo.py

```diff
@@ -203,15 +199,16 @@ def _filter_labels(
             return labels
         return labels
 
-    @staticmethod
     def _export(
+        self,
         image: ee.Image,
         region: ee.Geometry,
         filename: str,
         description: str,
         drive_folder: Optional[str] = None,
         dest_bucket: Optional[str] = None,
         file_dimensions: Optional[int] = None,
+        test: bool = False,
```

Comment:
Added `test` parameter here to help determine when `tifs` folder should be used in google cloud storage

Reply:
We are now coupling this exporter to an expected file layout in google cloud storage. Is there a way to document this somewhere for others to use?